Change style

assafecohen opened the pull request.

cohenassafe committed:
Change style

segalzeyalz commented on CheatSheet.md:
```diff
@@ -1,6 +1,7 @@
 Hey all put some links that can help us all in this project! 
 
 ##### GITHUB - 
+
```
The line wasn't added

cohenassafe committed:
Added new cheatSheet 

if someone find CheatSheet for react Replace " ADD SOME LINK HERE " to something real , Thanks.

segalzeyalz reviewed (approved):
goodjob

assafecohen reviewed (commented):
Accepted
